lib/restrictions.js

```diff
@@ -50,13 +56,26 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
  * @returns {Promise<object>} An object with the inserted record.
  */
 export async function insert({restriction} = {}) {
+  await validateSchema({restriction});
   const collection = database.collections['resource-restriction-restriction'];
   const now = Date.now();
   const meta = {created: now, updated: now};
   let record = {
     meta,
     restriction
   };
+  if(restriction.method === 'limitOverDuration') {
+    const duration = restriction.methodOptions.duration;
+    // pattern is ISO8601 regex
+    if(!(duration && pattern.test(duration))) {
+      throw new BedrockError(
+        `Duration '${duration}' is not valid.`,
```

Comment:
Double quotes are typically used for this instead of single quotes.  Just by convention.  It's highly likely that we have not been 100% consistent with this.